src/main/java/dev/Roach/AlertService.java

```diff
@@ -0,0 +1,85 @@
+package dev.Roach;
+
+import dev.Roach.datamodel.game.Game;
+import dev.Roach.datamodel.game.GamePojo;
+import dev.Roach.fetchers.GamesFetcher;
+import lombok.Setter;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+@Setter
+public class AlertService {
+
+    private HttpClient httpClient = HttpClient.newBuilder().build();
+
+    private GamesFetcher gamesFetcher = new GamesFetcher();
```

Comment:
final

Reply:
Cannot really make it final because my tests are based on this + setter from lombok. If I make it final I can't use setter. Was thinking about making it static but I don't know if it is a good idea ?